www/front_src/src/Resources/Details/tabs/Services/Graphs.tsx

```diff
@@ -63,7 +63,7 @@ const ServiceGraphs = ({
           const isLastService = equals(last(servicesWithGraph), service);
 
           return (
-            <div key={id}>
+            <React.Fragment key={id}>
```

Comment:
The fragment creates a weird behaviour on Firefox when scrolling through the list (blinking), that's why a div has been placed here

Reply:
Let's take care about Firefox 👀

Reply:
Event if I don't have this issue using FF 78

Reply:
I love FF ^_^'